middleware/index.ts

```diff
@@ -18,6 +17,7 @@ import {
     Org,
     User,
 } from '@salesforce/salesforce-sdk/dist/functions';
+import * as path from "path";
```

Comment:
looks like we only use `path.join` below if it helps to minimize the import surface area.